lms/lms/utils.py

```diff
@@ -156,15 +156,15 @@ def get_tags(course):
 def get_instructors(course):
 	instructor_details = []
 	instructors = frappe.get_all(
-		"Course Instructor", {"parent": course}, ["instructor"], order_by="idx"
+		"Course Instructor", {"parent": course}, ["instructor"], order_by="idx", pluck="instructor"
```

Comment:
@tahir-zaqout no need to have a fields array when you are plucking.

Reply:
Also please install pre-commit on your system. It will fix the linters before committing.